common/beerocks/btl/btl_uds.cpp

```diff
@@ -124,6 +125,14 @@ bool transport_socket_thread::bus_send(ieee1905_1::CmduMessage &cmdu, const std:
 
 bool transport_socket_thread::from_bus(Socket *sd) { return sd == bus; }
 
+bool transport_socket_thread::skip_filtered_message_type(ieee1905_1::eMessageType msg_type)
```

Comment:
This function should be removed completely and replaced with a static function in the btl_uds.cpp implementation only.

Reply:
That is not possible since it is being called on socket_thread.